jc/parsers/net_user.py

```diff
@@ -0,0 +1,441 @@
+                "user_name": "james"
+            },
+            {
+                "user_name": "krbtgt"
+            },
+            {
+                "user_name": "liam"
+            },
+            {
+                "user_name": "localadmin"
+            },
+            {
+                "user_name": "tiffany"
+            }
+        ]
+    }
+
+    $ net users Administrator | jc --net-user -p | jq
+    {
+        "domain": "",
+        "user_accounts": [
+            {
+                "account_active": true,
+                "account_expires": "Never",
+                "comment": "Built-in account for administering the computer/domain",
+                "country_region_code": "000 (System Default)",
+                "global_group_memberships": [],
+                "last_logon": "2024-08-23T13:47:11",
+                "local_group_memberships": [
+                    "Administrators"
+                ],
+                "logon_hours_allowed": "All",
+                "password_changeable": "2021-12-17T11:07:14",
+                "password_expires": "2022-01-27T11:07:14",
+                "password_last_set": "2021-12-16T11:07:14",
+                "password_required": true,
+                "user_may_change_password": true,
+                "user_name": "Administrators",
+                "workstations_allowed": "All"
+            }
+        ]
+    }
+
+    $ net users Administrator /domain | jc --net-user -p | jq
+    {
+        "domain": "somecompany.corp",
+        "user_accounts": [
+            {
+                "account_active": true,
+                "account_expires": "Never",
+                "comment": "Built-in account for administering the computer/domain",
+                "country_region_code": "000 (System Default)",
+                "global_group_memberships": [
+                    "Domain Admins",
+                    "Domain Users",
+                    "Group Policy Creator",
+                    "Enterprise Admins",
+                    "Schema Admins"
+                ],
+                "last_logon": "2024-07-17T13:46:12",
+                "local_group_memberships": [
+                    "Administrators"
+                ],
+                "logon_hours_allowed": "All",
+                "password_changeable": "2023-09-30T11:44:26",
+                "password_expires": "Never",
+                "password_last_set": "2023-09-29T11:44:26",
+                "password_required": true,
+                "user_may_change_password": true,
+                "user_name": "Administrators",
+                "workstations_allowed": "All"
+            }
+        ]
+    }
+"""
+
+
+from datetime import datetime
+import re
+import jc.utils
+
+
+class info():
+    """Provides parser metadata (version, author, etc.)"""
+    version = '1.0'
+    description = '`net user` command parser'
+    author = 'joehacksalot'
+    author_email = 'joehacksalot@gmail.com'
+    compatible = ['windows']
+    magic_commands = ['net-user']
```

Comment:
looks like this should be 'net users'

Reply:
Leaving as is. see [https://github.com/kellyjonbrazil/jc/pull/602#discussion_r1848938465](reference)

i am removing the '-'